spec/classes/init_spec.rb

```diff
@@ -865,6 +719,18 @@
         :invalid => ['./invalid',3,2.42,['array'],a={'ha'=>'sh'}],
         :message => 'is not an absolute path',
       },
+      'regex_file_mode' => {
+        :name    => ['crontab_mode', 'cron_allow_mode', 'cron_deny_mode', 'cron_dir_mode'],
+        :valid   => ['0755','0750'],
+        :invalid => ['invalid','755',0755,'0980',2.42,['array'],a={'ha'=>'sh'}],
```

Comment:
@Phil-Friderici I added '0980' to the list of invalid entries to pick up errors like using `[0-9]{4}` instead of `[0-7]{4}`. We should be careful to ensure that is used elsewhere in the future.